test/integration/decorators.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+declare function fixture(id: string): HTMLElement;
+
+suite('TypeScript Decorators', function() {
+
+  suite('@customElement', function() {
+
+    test('defines an element', function() {
+      var testElement = fixture('test-element');
+      chai.assert.instanceOf(testElement, TestElement);
```

Comment:
I am surprised that it is able to find the typings for chai automatically

Reply:
Any typings installed to the `@types` namespace (in this case `npm install @types/chai`) are automatically included in TypeScript compilation.

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types